sdk/python/feast/infra/registry/remote.py

```diff
@@ -48,21 +58,33 @@ def apply_entity(self, entity: Entity, project: str, commit: bool = True):
             entity=entity.to_proto(), project=project, commit=commit
         )
 
-        self.stub.ApplyEntity(request)
+        if self.auth_config.type is not AuthType.NONE.value:
```

Comment:
ca we at least simplify the whole block by moving the `type` check in the function and just call:
```py
self.stub.ApplyEntity(request=request, metadata=create_grpc_metadata(self.auth_config))
```
where `create_grpc_metadata` returns `None` if the type is `NONE`?